docs/_documentations/che-installinfo.md

```diff
@@ -14,39 +14,101 @@ Install Che to use with Codewind or prepare to use Codewind with an existing Che
+`kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.10.0/setup/install_che/codewind-clusterrole.yaml`
 
 ## Installing Che with chectl
 
+Codewind requires Eclipse Che to be installed with HTTPS in order to function properly. The following section, `Installing Che` will cover setting up Eclipse Che with TLS certificates.
+
+### Installing Che
 The fastest way to install Eclipse Che for Codewind is to use the `chectl` CLI. To install the `chectl` CLI tool, see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/).
 
-After you install `chectl`, complete the following steps:
-1. Download the [codewind-checluster.yaml](https://github.com/eclipse/codewind-che-plugin/blob/0.9.0/setup/install_che/che-operator/codewind-checluster.yaml) file.
-    - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
-2. If you're installing Che on a Kubernetes platform other than OpenShift, determine your Ingress domain.
+After you install `chectl`, download the [codewind-checluster.yaml](https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.10.0/setup/install_che/che-operator/codewind-checluster.yaml) file
+ - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
+
+**Installing on OpenShift:**
+
+Eclipse Che on OpenShift makes use of the router's existing certificates. 
+Run the following command to install Che on OpenShift with Chectl: 
+   ```
+   $ chectl server:start --platform=openshift --installer=operator --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.9.0
+   ```
+
+**Installing on Kubernetes:**
+
+1. Create the `che` namespace if it doesn't already exist: `kubectl create namespace che`
+2. Determine your Ingress domain.
     - Set the `spec.server.ingressDomain` field in the Che `.yaml` file to the Ingress domain.
     - If you're unsure of your Ingress domain, ask your cluster administrator.
-3. Install Che:
-    - On OpenShift, run the following command: `chectl server:start --platform=openshift --installer=operator --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.5.1`
-    - On Kubernetes, run the following command: `chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.5.1`
+3. Generate TLS certificates and keys according to https://www.eclipse.org/che/docs/che-7/setup-che-in-tls-mode-with-self-signed-certificate/#generating-self-signed-certificates_setup-che-in-tls-mode-with-self-signed-certificate
```

Comment:
Let's refer to the file name instead of the URL:
`3. Generate TLS certificates and keys. For more information, see [Generating self-signed TLS certificates](https://www.eclipse.org/che/docs/che-7/setup-che-in-tls-mode-with-self-signed-certificate/#generating-self-signed-certificates_setup-che-in-tls-mode-with-self-signed-certificate).`